### PR TITLE
practice-3

### PR DESCRIPTION
**Pull request by NastasiaIOSdev**

### CODE REVIEW

File: TaskTwoV1/Cats/CustomCells/HalfWidthTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    func configure(with viewModel: CellViewModel) {
+        breedLabel.text = viewModel.title
+        countryLabel.text = viewModel.subtitle
+
+        if let data = viewModel.imageData {
+            catUIImageView.image = UIImage(data: data)
+        }
+        else if let url = viewModel.imageUrl {
+            URLSession.shared.dataTask(with: url) { [weak self] data, _, error in
+                guard let data = data, error == nil else {
```
Comment (vasilevkin) [inline TaskTwoV1/Cats/CustomCells/HalfWidthTableViewCell.swift:49]: better use  let error = error and guard let
try do not use nil comparison in direct way ( == nil )

**Review by vasilevkin — CHANGES_REQUESTED** [review]
it looks better, proceed with additional tasks or next practice
you are on the correct way!

**Comment by NastasiaIOSdev** [comment]
Thank you for the good words, Sergey)))!!

File: TaskTwoV1/Cats/CustomCells/HalfWidthTableViewCell.swift (hunk 1 above)
Comment (NastasiaIOSdev) [reply TaskTwoV1/Cats/CustomCells/HalfWidthTableViewCell.swift:49]: fix

**Comment by NastasiaIOSdev** [comment]
Fixed all comments

**Review by vasilevkin — APPROVED** [review]

**Pull request merged**